data/brands/amenity/parcel_locker.json

```diff
@@ -143,6 +154,17 @@
         "parcel_mail_in": "yes"
       }
     },
+    {
+      "displayName": "DPD Pickup Station",
+      "id": "dpdpickupstation-0154ac",
+      "locationSet": {"include": ["pl"]},
+      "tags": {
+        "amenity":"parcel_locker",
+        "brand":"DPD Pickup Station",
+        "operator:wikidata":"Q541030",
+        "operator":"DPD"
```

Comment:
looks like this item don't have `brand:wikidata` but a `operator:wikidata`, and is this really don't need `parcel_pickup` key?

Reply:
As not all of them currently mapped in Poland had = I didn't want to add it, however [official website](https://www.dpd.com/pl/pl/automaty-paczkowe-w-dpd-polska/) says clearly they should be able to meet criteria. I will update my PR with that change.

There is no wikidata page about that brand, however brand name is in use, so I believe proposed tagging is ok. `operator:wikidata` refers to DPDgroup, as it's operating service. What is wrong here?

Here is example photo of such locker:

![photo_2022-09-27_22-56-50](https://user-images.githubusercontent.com/6720051/192633828-0f09482b-0b6b-4765-9191-fec9304a5e99.jpg)

Reply:
Ok, I decided it's best to add Wikidata item.
I updated template with new brand:wikidata reference.